src/components/StringField.vue

```diff
@@ -22,15 +22,43 @@
                 :state="isValid"
                 :trim="true"
             />
+            <div class="mt-2" v-if="city">Ce code INSEE correspond à {{ this.city }}.</div>
         </b-form-group>
     </div>
 </template>
 <script>
+import { EventBus } from '@/event-bus.js'
 import ValidateField from '@/mixins/ValidateField.vue'
 
 export default {
     name: 'StringField',
     mixins: [ValidateField],
+    data() {
+        return {
+            city: null
+        }
+    },
+    mounted() {
+        EventBus.$on('field-value-changed', (field_name, value) => {
+            if (this.field.name !== field_name) return
+            if (this.isInsee) {
+                this.handleInseeInput(value)
+            }
+        })
+    },
+    methods: {
+        handleInseeInput: function(value) {
+            fetch(`https://geo.api.gouv.fr/communes/${value}`).then(r => {
```

Comment:
Not sure this is the proper way to handle 400 / 404 since `value` will be not valid sometimes when the user begins to type